kbase-extension/static/kbase/js/widgets/appWidgets2/validation.js

```diff
@@ -450,6 +452,9 @@ define([
             } else if (options.invalidValues && options.invalidValues.has(value)) {
                 diagnosis = Constants.DIAGNOSIS.INVALID;
                 errorMessage = options.invalidError ? options.invalidError : 'value is invalid';
+            } else if (options.validValues && !options.validValues.has(value)) {
```

Comment:
I wonder if LGTM or one of the other code checkers will suggest merging this with the option above, i.e.

```js
} else if (options.invalidValues && options.invalidValues.has(value) 
    || options.validValues && !options.validValues.has(value)) {
        diagnosis = Constants.DIAGNOSIS.INVALID;
        errorMessage = options.invalidError ? options.invalidError : 'value is invalid';
}

Reply:
Ah, I didn't catch that. Thanks!